CardData.py

```diff
@@ -181,62 +184,66 @@ def _createBaseCardJson(self):
 
                 art = {}
                 for imageSize in IMAGE_SIZES:
```

Comment:
loop variables still camel-cased

Reply:
AH, I thought this was accepted. Alright, give me a sec.